Content.Client/DeltaV/CartridgeLoader/Cartridges/CrimeAssistUiFragment.xaml.cs

```diff
@@ -0,0 +1,180 @@
+            CrimeAssistUiState.UiStates.Result_GrandTheft => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_BlackMarketeering => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Sabotage => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Mindbreaking => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Assault => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_AbuseOfPower => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Possession => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Endangerment => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_BreakingAndEntering => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Rioting => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_ContemptOfCourt => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_PerjuryOrFalseReport => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_ObstructionOfJustice => "[color=#7b5430]" + Loc.GetString("crime-assist-crimetype-felony") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Murder => "[color=#7b2e30]" + Loc.GetString("crime-assist-crimetype-capital") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Terrorism => "[color=#7b2e30]" + Loc.GetString("crime-assist-crimetype-capital") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_GrandSabotage => "[color=#7b2e30]" + Loc.GetString("crime-assist-crimetype-capital") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Decorporealisation => "[color=#7b2e30]" + Loc.GetString("crime-assist-crimetype-capital") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Kidnapping => "[color=#7b2e30]" + Loc.GetString("crime-assist-crimetype-capital") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_Sedition => "[color=#7b2e30]" + Loc.GetString("crime-assist-crimetype-capital") + "[/color]",
+            CrimeAssistUiState.UiStates.Result_SexualHarrassment => "[color=#7b2e30]" + Loc.GetString("crime-assist-crimetype-capital") + "[/color]",
+            _ => ""
```

Comment:
It'd be nice if this functioned via yaml or something, as if we ever change splaw the only way to update this is by touching code

Reply:
I would need to learn a bit more about yml for that, but it does sound like a big improvement. I should have time to look into this saturday :)

Reply:
It is done! Learnt a lot about prototypes while doing it, too